src/main/java/org/kaiaccount/account/inter/type/player/AbstractPlayerAccount.java

```diff
@@ -135,6 +135,19 @@ public CompletableFuture<TransactionResult> multipleTransaction(
         return account;
     }
 
+    public @NotNull boolean deleteBankAccount(@NotNull String name) {
```

Comment:
Mind just annotating this with a ``@Override`` to just make it clear its coming from a method outside the class and that the class is modifying the behaviour